src/dist/client_auth.rs

```diff
@@ -627,15 +840,19 @@ pub fn get_token_oauth2_implicit(client_id: &str, mut auth_url: Url) -> Result<S
         shutdown_tx: Some(shutdown_tx),
     };
     *implicit::STATE.lock().unwrap() = Some(state);
-    let shutdown_signal = shutdown_rx.map_err(|e| {
-        warn!(
-            "Something went wrong while waiting for auth server shutdown: {}",
-            e
-        )
-    });
+    let shutdown_signal = shutdown_rx;
 
     let mut runtime = Runtime::new()?;
-    runtime.block_on(server.with_graceful_shutdown(shutdown_signal))?;
+    runtime.block_on(server.with_graceful_shutdown(async move {
+        let _ = shutdown_signal;
+    }))
+    // .map_err(|e| {
```

Comment:
do you want to leave this code commented?

is it ok to not handle this in new code?

Reply:
Its a bit of a TODO..